src/components/Facets.tsx

```diff
@@ -0,0 +1,88 @@
+import { Facets as FacetsCompoundComponent } from './Filters/Facets';
+import { FilterGroup } from './FilterGroup';
+import { CompositionMethod } from '../hooks/useComposedCssClasses';
+import { Fragment } from 'react';
+
+/**
+ * The CSS class interface for {@link Facets}.
+ *
+ * @public
+ */
+export interface FacetsCssClasses {
+  container?: string,
+  divider?: string,
+  searchInput?: string,
+  optionsContainer?: string,
+  option?: string,
+  optionInput?: string,
+  optionLabel?: string
+}
+
+/**
+ * Props for the {@link Facets} component.
+ *
+ * @public
+ */
+export interface FacetsProps {
+  /** Whether or not the filter is collapsible. Defaults to true. */
+  collapsible?: boolean,
+  /**
+   * If the filter group is collapsible, whether or not it should start out
+   * expanded. Defaults to true.
+   */
+  defaultExpanded?: boolean,
+  /** Whether or not to display a text input to search for filter options. */
+  searchable?: boolean,
+  /**
+   * Whether or not a search is automatically run when a filter is selected.
+   * Defaults to true.
+   */
+  searchOnChange?: boolean,
+  /** CSS classes for customizing the component styling. */
+  customCssClasses?: FacetsCssClasses,
+  /** {@inheritDoc CompositionMethod} */
+  cssCompositionMethod?: CompositionMethod
+}
+
+/**
+ * A component that displays all facets applicable to the current vertical search.
+ *
+ * @param props - {@link FacetsProps}
+ * @returns A React component for facets
+ *
+ * @public
+ */
+export function Facets(props: FacetsProps) {
+  const { searchOnChange, customCssClasses = {}, ...filterGroupProps } = props;
+  const { container:containerCssClasses, divider, ...filterGroupCssClasses } = customCssClasses;
```

Comment:
what's the reason for renaming the container class?

Reply:
just to be more specific that this is cssClasses

Reply:
I thought we used cssClasses when there was more than one class, and here it's just container?

Reply:
we usually have `blahCssClasses.container` or `containerCssClasses`, so the rename is so it's a bit more clear that this isn't a container element or anything but a string of css classes. If you feel strongly about it, I can update it to just `container`

Reply:
I think maybe it would be better to name it `containerClassName` so it still makes it clear that it's a string for styling and not an element, but also make it clear that it's a single class name and not an object with multiple classes.

in the same way, we should probably change the divider CSS class name also. and maybe in the Divider component, we can change the prop name to `className` instead of `cssClasses` to indicate it's a single class and not an object

Reply:
sure